docs/deploying-helium.md

```diff
@@ -109,7 +140,7 @@ $ az group create -l eastus -n {app_prefix}helium
 The Azure Container Registry (ACR) is where Docker container images are stored. This infrastructure can be created in the following manner:
 
 ```bash
-$ az acr create --name {app_prefix}heliumacr --resource-group {app_prefix}helium --sku Basic --admin-enabled true --location eastus
+$ az acr create -n $acrName --resource-group $resourceGroupName --sku Basic --admin-enabled false -l $location
```

Comment:
I updated this to admin-enabled false, but am not able to successfully deploy the container image from the portal in a later step.

Reply:
When you setup the repo, you'll have to specify the SP ID and key as part of the setup. There's a bug in Ibiza where this won't work but it can be done from the CLI.

You will want to create a SP with pull (or read?) only access to the ACR and ideally an SP with read/write and an SP with admin so that you can use least privilege SPs.

Reply:
az webapp config container set -g $rg -n $app -i helium:0.1.0 -r https://gbsazworkshop2019commonacr.azurecr.io -u $acr -p $acr_pw

$acr should be the SP ID
$acr_pw should be the SP Token

Reply:
I made some changes to add in SPs with different roles -- it worked for me once, but i'm going to run through it again to make sure it repros well.  Let me know if the changes I made dont look quite right.